### DISCUSSION TITLE
how can i convert Autokeras model into Keras model?

### DISCUSSION THREAD

**Discussion by XYshield** [keras-team/autokeras#1490] [Q&A]
![屏幕截图(201)](https://user-images.githubusercontent.com/77934391/105639143-d3f2f900-5e76-11eb-828c-790187095779.png)
i tried to clone a autokeras model to keras, and train this keras model and then saved as h5 file,but i failed.(save or restore weightsthat is not in an instance of 'tf.variable' is not supported in h5) i don't know why. could someone give me some advises?  thanks!

**Comment by haifeng-jin** [comment]
When you save the model, use a file name that is NOT ended with ".h5".
```python
model = automodel.export_model()
model.save('model') # instead of model.save('model.h5')
```
